doc/api/fs.md

```diff
@@ -4245,6 +4245,32 @@ On Linux, positional writes do not work when the file is opened in append mode.
 The kernel ignores the position argument and always appends the data to
 the end of the file.
 
+#### filehandle.writev(buffers[, position])
```

Comment:
It seems this section should be placed after the `#### filehandle.writeFile()` alphabetically.